skactiveml/base.py

```diff
@@ -322,12 +325,22 @@ def _validate_data(
         ) = super()._validate_data(
             X, y, candidates, batch_size, return_utilities, reset, check_X_dict
         )
-        y = column_or_1d(y, warn=True)
+
+        y = column_or_1d(y, warn=True) if not is_multilabel else check_array(y, ensure_2d=True, force_all_finite="allow-nan")
+        # here check
+        if is_multilabel and not np.all(np.isin(y, [0, 1]) | np.isnan(y)): # possible 0,1-encoded check
```

Comment:
- Usage of is_unlabeled with self.missing_label
- Unlabeled instances must contain only missing_label for every class
- Raise Error in case of inconsistencies